packages/e2e-tests/specs/blocks/list.test.js

```diff
@@ -126,9 +127,7 @@ describe( 'List', () => {
 		await page.keyboard.press( 'Enter' );
 		// Pointer device is needed. Shift+Tab won't focus the toolbar.
```

Comment:
Should those comments be removed? I don't quite understand why they are here without a broader context.